examples/fluids/navierstokes.c

```diff
@@ -533,8 +533,8 @@ static PetscErrorCode ComputeLumpedMassMatrix(Ceed ceed, DM dm,
   PetscFunctionReturn(0);
 }
 
-PetscErrorCode SetUpDM(DM dm, problemData *problem, PetscInt degree,
-                       SimpleBC bc, void *ctxSetup) {
+static PetscErrorCode SetUpDM(DM dm, problemData *problem, PetscInt degree,
```

Comment:
I've thought about suggesting that we move these functions into their own file to make it a little bit easier to find things, like we did with the solids code. I'm not sure if 1) other people like that and 2) other people find it valuable.

Reply:
Definitely. This code needs some major refactoring and separation of modules. I was leaving that to subsequent PRs as we were discussing on Zulip. For now this PR was mainly focused with fixing impelling issues like broken pieces and functionalities.